pkgs/applications/science/machine-learning/vowpal-wabbit/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchgit, cmake, boost169, zlib }:
+
+let
+  version = "8.9.2";
+in
+stdenv.mkDerivation {
+  pname = "vowpal-wabbit";
+  version = "${version}";
+  src =
+    fetchgit {
+      url = "https://github.com/VowpalWabbit/vowpal_wabbit.git";
+      rev = "${version}";
```

```suggestion
        rev = version;
```
String interpolation is not required here.

---

Will fix